test/evaluation/evaluationClientWrapper.spec.ts

```diff
@@ -288,7 +288,7 @@ describe('evaluate', () => {
     delete pdSchema!.submission_requirements;
     vpSimple!.verifiableCredential![0]!.vc!.issuer = 'did:foo:123';
     const evaluationClientWrapper: EvaluationClientWrapper = new EvaluationClientWrapper();
-    vpSimple!.holder = evaluationClientWrapperData.getHolderDID()[0];
+    vpSimple!.holder = [evaluationClientWrapperData.getHolderDID()[0]];
```

Comment:
seems redundant. `evaluationClientWrapperData.getHolderDID()` returns a single entry list

Reply:
It cannot be a list, it's inside a presentation. Fixed!